test/opt/loop_optimizations/fusion_pass.cpp

```diff
@@ -17,18 +17,14 @@
 #include "gmock/gmock.h"
 #include "test/opt/pass_fixture.h"
 
-#ifdef SPIRV_EFFCEE
 #include "effcee/effcee.h"
```

Comment:
Can you remove the blank line before the include and then hopefully the formatter should order them correctly (this and other files)